crates/cli/src/config.rs

```diff
@@ -1,11 +1,22 @@
 use crate::util::{contains_protocol, host_or_url_to_host_and_protocol};
 use anyhow::Context;
 use jsonwebtoken::DecodingKey;
-use serde::{Deserialize, Serialize};
+use spacetimedb::config::{set_opt_value, set_table_opt_value};
 use spacetimedb_fs_utils::atomic_write;
 use spacetimedb_paths::cli::CliTomlPath;
-
-#[derive(Serialize, Deserialize, Clone, Debug)]
+use std::collections::HashMap;
+use toml_edit::{ArrayOfTables, DocumentMut, Item, Table};
```

Comment:
I would prefer to see these, especially `Table`, used by full path or with a qualified import. `Item` and `Table` in the code below are at best unhelpful names, and at worst actively confusing, given that we use the word "Table" to mean something very different in most of our code. I would do `use toml_edit as toml;` and then write `toml::Table`, `toml::DocumentMut`, &c. below.